dash/include/dash/halo/Halo.h

```diff
@@ -1360,14 +1594,16 @@ class HaloMemory {
    * \return Pointer to the first halo element or nullptr if the
    *         region doesn't exist
    */
-  Element_t* pos_at(region_index_t index) { return _halo_offsets[index]; }
+  Element_t* first_element_at(region_index_t index) {
+    return _halo_offsets[index];
+  }
 
   /**
    * Pointer to the first halo element
    *
    * \return Pointer to the first halo element
    */
-  Element_t* pos_begin() { return _halobuffer.data(); }
+  Element_t* first_element() { return _halobuffer.data(); }
```

Comment:
See above

Reply:
Normally you want a pointer, to iterate through the data. More interesting would be the return of an iterator. What you think?